scripts/functions/manage/base_fetch

```diff
@@ -248,6 +263,8 @@ __rvm_fetch_from_git_revision_or_sha()
   then __from="${rvm_ruby_sha}"
   elif [[ -n "${rvm_ruby_repo_branch:-}" ]]
   then __from="${rvm_ruby_repo_branch}"
+  elif [[ -n "${rvm_ruby_repo_tag:-}" ]]
+  then __from="${rvm_ruby_repo_tag}"
```

Comment:
how this is different from the earlier line `__from="${rvm_ruby_tag#t}"`? why can't we use that?